OpenRA.Mods.Common/Activities/Attack.cs

```diff
@@ -78,7 +78,7 @@ protected virtual Activity InnerTick(Actor self, AttackBase attack)
 				return NextActivity;
 
 			var type = Target.Type;
-			if (!Target.IsValidFor(self) || type == TargetType.FrozenActor)
+			if (!Target.IsValidFor(self))
 				return NextActivity;
 
 			if (attack.Info.AttackRequiresEnteringCell && !positionable.CanEnterCell(Target.Actor.Location, null, false))
```

Comment:
@pchote - I believe I fixed this (it seems like there was only one place that needed fixing in the method, the rest either check the type before accessing the actor field, or use CenterPosition)